lib/storage/http.js

```diff
@@ -26,7 +26,7 @@ async function download(asset, file) {
         await http.downloadFileConcurrently(asset.url, file, {
             retryEnabled: !process.env.ASSET_COMPUTE_DISABLE_RETRIES,
             headers: asset.headers,
-            contentType: asset.type || asset.mimeType,
+            contentType: asset.type || asset.mimetype || asset.mimeType,
```

Comment:
unfortunately, we pass the `assetReference`, not the `Asset` class instance to the download function, so we need to check all places here
https://github.com/adobe/asset-compute-pipeline/blob/master/lib/storage/index.js#L75

Reply:
Since `getAsset` is getting the Asset, it make sense to "only" have the asset reference there.